test/sign_in_users.spec.js

```diff
@@ -0,0 +1,132 @@
+const supertest = require('supertest');
```

Comment:
Generally, we test
/controllers
/services
/interactors

and not a function specific. 
You should move these tests to an unique file call, for example:
/controllers/users.spec.js

(sign up tests too)